docs/peg_oled_display.md

```diff
@@ -0,0 +1,109 @@
+# Peg Oled Display
+To use this you need to make some changes to your kb.py as well as you main.py I will break it into two sections for you. 
+
+### What you can and cant do
+
+#### Can do
+* display images

```

Comment:
I'm being picky, but can we capitalize the lists? It just looks better, and is more consistent with the rest of the docs I believe. This applies to all start of lines that start a new sentence, or bullet point.

Reply:
good catch!